src/app/(layout-with-navigation)/(simulation)/amis/invitation/_components/LaconicRanking.tsx

```diff
@@ -0,0 +1,79 @@
+import Trans from '@/components/translation/Trans'
+import Emoji from '@/design-system/utils/Emoji'
+import { Group } from '@/types/groups'
+
+type Props = {
+  group: Group
+}
+
+export default function LaconicRanking({ group }: Props) {
+  // If only one participant
+  if (group.participants.length !== 1) {
+    return (
+      <section className="mt-6 rounded-xl bg-primary-50 p-4">
+        <p className="mb-0">
+          <Trans>Qui de </Trans>{' '}
+          <strong className="text-primary-700">
+            {group.participants[0].name}
+          </strong>{' '}
+          <Trans>et vous aura la plus faible empreinte ?</Trans>{' '}
+          <Emoji>🤓</Emoji>
+        </p>
+      </section>
+    )
+  }
+
+  const particpantsOrdered = group.participants.sort((a, b) => {
+    if (!a.simulation.computedResults || !b.simulation.computedResults) {
+      return 0
+    }
+
+    if (
```

Comment:
`return a.simulation.computedResults.bilan < b.simulation.computedResults.bilan ? -1 : 1` ?

Reply:
c'est Copilot qui te l'a soufflé ? Pourquoi ne pas couvrir le cas 0

Reply:
I'm gonna trust you on this

Reply:
modifié